src/concretize.rs

```diff
@@ -200,33 +200,35 @@ pub struct WolfSSL {
 
 impl Stream for WolfSSL {
     fn add_to_inbound(&mut self, result: &OpaqueMessage) {
-       todo!() // self.stream.get_mut().add_to_inbound(result)
+        let a = self.stream.get_mut();
+        a.add_to_inbound(result)  // SEGFAULT: here a = NULL when we execute the first input, which triggers add_to_inbound here! Somehow, bio initialization fails !
```

Comment:
Segfault here when running `cago test --package tlspuffin --lib tests::seeds::test_seed_client_attacker12 --no-default-features --features wolfssl520 -- --exact`

Reply:
Feel free to tell me if you need support here :)
I'm quite busy with another project right now, but can always support.